CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+# See https://github.blog/2017-07-06-introducing-code-owners/ for details
+# Format of the file follows: https://git-scm.com/docs/gitattributes
+
+# Fallback to core group for crates or folders without specific owners, like docs/.
+
+* @nciric @zbraniecki @sffc @echeran
+
+# List of components with at least two owners as specified in docs/crate-ownership.
+
+components/locale/*               @zbraniecki @nciric
```

Comment:
I think you don't want the `*` at the end.  From https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners:

> ```
> # The `docs/*` pattern will match files like
> # `docs/getting-started.md` but not further nested files like
> # `docs/build-app/troubleshooting.md`.
> docs/*  docs@example.com
> ```

Reply:
good catch